src/node_os.cc

```diff
@@ -176,6 +176,13 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
     fields[field_idx++] = ci->cpu_times.irq;
     model_argv[model_idx++] = OneByteString(env->isolate(), ci->model);
 
+#if ENABLE_TTD_NODE
+    if(s_doTTRecord || s_doTTReplay) {
```

Comment:
Can you fix this lint issue?

src\node_os.cc:180:  Missing space before ( in if(  [whitespace/parens] [5]